hack/k8s/spike-install.sh

```diff
@@ -53,24 +53,15 @@ echo "SPIKE namespaces:"
 kubectl get namespaces | grep spike || echo "No spike namespaces found"
 
 helm upgrade --install -n spire-mgmt spire-crds spire-crds \
-  --repo https://spiffe.github.io/helm-charts-hardened/ --create-namespace
+  --repo https://spiffe.github.io/helm-charts-hardened/ \
+  --version 0.26.1 --create-namespace
```

Comment:
I pinned versions, just in case. but we should remember to update and test with newer versions too.